README.md

```diff
@@ -26,20 +26,28 @@ This is the recommended method if you already have a collection of your own Test
 
 ## Configure the Engine
 
-The engine can be configured through several variables, each of which has a preset value that can be modifed during runtime:
-
-- `TEST_SERVER_URL`: The endpoint against which the runnables will be executed.
-- `LOAD_NON_FHIR_FIXTURES`: Whether to ignore non-FHIR fixtures. Non-FHIR fixtures are not currently supported by the [spec](https://build.fhir.org/testscript.html), however we recognize several use cases where they would be valuable. Expects [T/F].
-- `TESTSCRIPT_PATH`: The relative path to the directory containing the TestScript resources (as JSON or XML) to be executed by the engine. If any TestScript in the directory uses a fixture, the directory MUST also include a `fixtures` subfolder containing files whose relative paths match the reference value of a fixture within a TestScript.
-- `TESTREPORT_PATH`: The relative to the directory containing the TestReports output following their partner TestScript execution.
+The engine can be configured through three methods: config.yml, commandline arguments, and interactive mode.
 
 ### Commandline Arguments
 
-Some configurations can be set using command line arguments:
-- `-r` or `--runnable`: the next argument must be the name of runnable to execute (from `TestScript.name`). Only works if the noninteractive flag is provided. 
-- `-n` or `--noninteractive`: disable confirmation of configuration settings
+Command line arguments can be used when starting the engine with the following format:
+`bundle exec bin/testscript_engine option [OPTIONS]`
+
+[OPTIONS]
+- `--interactive`: run the engine on interactive mode
+- `--config`: run the engine on YAML configuration file. Only works on non-interactive mode. The next argument(optional) must be the name of YAML file. Without it, the engine intakes default YAML file (config.yml) 
```

Comment:
maybe `--config [CONFIG_FILE_PATH]` to make that next piece of input more clear. Also I would say that the following file path should be REQUIRED. You want to avoid being able to say the same thing multiple ways and by making it optional a command without `--config` and with `--config` without a specific path are the same, right?

Lastly, examples of actual commands would be helpful.  I like they way they recommend doing it (here)[https://developers.google.com/style/code-syntax#example-command]; not exhaustive, but enough to give you a sense for the pattern (their syntax is a bit different, which is fine, I'm talking about the documentation style only)

Reply:
> 

Good comments. I will add them. Thanks!